### DISCUSSION TITLE
training an environment or area

### DISCUSSION THREAD

**Discussion by rycossette** [TheLastBen/fast-stable-diffusion#1616] [General]
I couldn't find anything existing about this...

I was thinking about training an environment, like my condo, with the various rooms etc. Is this even possible? I've only seen people, objects and styles, but nothing on something like this.

Has anyone tried this? If so what are your settings used to train?

**Comment by TheLastBen** [comment]
yes you can, make sure the instance images share something in common (style, design ...)

**Reply by rycossette** [reply]
So if I wanted to do one environment, with different areas, would I need to add certain captions to them? So I could call up prompts like, Condo, balcony, front, or condo, bathroom, shower. And get specific results.

**Reply by TheLastBen** [reply]
no need for captions, just use on unique unknown identifier

**Reply by rycossette** [reply]
For unique Identifiers, is it better to use something random for each one? like: asdgs3, gjfd, etc or viewa, viewb, viewc etc?

**Reply by TheLastBen** [reply]
if it's the same style, use the same identifier for everything

**Reply by rycossette** [reply]
I guess that's my question, it's hard to think of it as a style, it's different angles and perspectives of the an area... but if I want a specific angle to come up when I prompt i'm not sure what works best...

Something like Condo, living room, high - would give me a high angle view of the living room, 
condo, living room, back - would give me a view of the living room but looking towards the kitchen

Just an example of what I'm hoping to do
